Assets/MixedRealityToolkit/_Core/Definitions/InputSystem/MixedRealityInputSystemProfile.cs

```diff
@@ -74,6 +76,24 @@ public MixedRealityPointerProfile PointerProfile
         /// </summary>
         public IMixedRealitySpeechSystem SpeechSystem => speechSystem ?? (speechSystem = MixedRealityToolkit.Instance.GetService<IMixedRealitySpeechSystem>());
 
+
+        /// <summary>
+        /// The list of cultures where speech recognition is supported.  Cultures added as needed
+        /// </summary>
+        private List<CultureInfo> supportedVoiceCultures = new List<CultureInfo>
```

Comment:
I feel this should be configurable in the editor, as to which speech cultures are supported by the developers project.